test/integration/faraday_middleware_test.rb

```diff
@@ -22,6 +22,30 @@ def teardown
       Circuitbox.reset
     end
 
+    def open_circuit
```

Comment:
It would be good to hide that method on the private api of the test if want to use it only in the scope of this test. If we need to use it as a helper method then best place would be the test_helper.